lib/elixir/lib/kernel/parallel_compiler.ex

```diff
@@ -145,8 +148,11 @@ defmodule Kernel.ParallelCompiler do
         each_long_compilation: Keyword.get(options, :each_long_compilation, fn _file -> :ok end),
         each_module: Keyword.get(options, :each_module, fn _file, _module, _binary -> :ok end),
         beam_timestamp: Keyword.get(options, :beam_timestamp),
-        output: output,
         long_compilation_threshold: Keyword.get(options, :long_compilation_threshold, 15),
+        profile: Keyword.get(options, :profile),
+        cycle_start: System.monotonic_time(:millisecond),
```

Comment:
I think it is more precise to compute it as native and convert to millisecond only when displaying.